From 3ebb2bd9dd03701f24ccfae701ed5138e45e5bf6 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Wed, 28 Sep 2016 16:07:56 +0300
Subject: [PATCH 0531/1345] dts: mvpp2x: remove SerDes remap from cp110 dts
 files

commit  143ae74d4d2f860c92a7cb6b1c61a05e16a6ce57 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- SerDes resources shouldn't be ioremapped by driver

Change-Id: Ic7587de5d16d4f44b6a5b6ac6cc8309f1a61d744
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/33184
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi |    3 +--
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi   |    3 +--
 2 files changed, 2 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
index 3c4f5b9..7ac125d 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-1.dtsi
@@ -257,7 +257,6 @@ gop {
 ppv22@000000 {
 	compatible = "marvell,mv-pp22";
 	reg = <0x000000 0x90000>, /* Packet Processor regs */
-	      <0x120000 0x9000>, /* SERDES regs */
 	      <0x129000 0x0600>, /* XMIB regs */
 	      <0x12a000 0x200>,  /* LED regs */
 	      <0x12a200 0x200>,  /* SMI regs */
@@ -270,7 +269,7 @@ ppv22@000000 {
 	      <0x130e00 0x100>,  /* MSPG - GMAC regs */
 	      <0x130f00 0x100>,  /* MSPG - XLG MAC regs */
 	      <0x441000 0x1000>;  /* RFU-1 Regs */
-	reg-names = "pp", "serdes", "xmib", "led", "smi", "tai", "xsmi", "mg", "mspg", "xpcs",
+	reg-names = "pp", "xmib", "led", "smi", "tai", "xsmi", "mg", "mspg", "xpcs",
 		    "fca", "gmac", "xlg", "rfu1";
 	clocks = <&cps_syscon0 1 3>, <&cps_syscon0 1 18>, <&cps_syscon0 1 9>,
 		 <&cps_syscon0 1 6>, <&cps_syscon0 1 5>;
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index 13cf70e..d3d3247 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -343,7 +343,6 @@ gop {
 ppv22@000000 {
 	compatible = "marvell,mv-pp22";
 	reg = <0x000000 0x90000>, /* Packet Processor regs */
-	      <0x120000 0x9000>, /* SERDES regs */
 	      <0x129000 0x0600>, /* XMIB regs */
 	      <0x12a000 0x200>,  /* LED regs */
 	      <0x12a200 0x200>,  /* SMI regs */
@@ -356,7 +355,7 @@ ppv22@000000 {
 	      <0x130e00 0x100>,  /* MSPG - GMAC regs */
 	      <0x130f00 0x100>,  /* MSPG - XLG MAC regs */
 	      <0x441000 0x1000>;  /* RFU-1 Regs */
-	reg-names = "pp", "serdes", "xmib", "led", "smi", "tai", "xsmi", "mg", "mspg", "xpcs",
+	reg-names = "pp", "xmib", "led", "smi", "tai", "xsmi", "mg", "mspg", "xpcs",
 		    "fca", "gmac", "xlg", "rfu1";
 	clocks = <&cpm_syscon0 1 3>, <&cpm_syscon0 1 18>, <&cpm_syscon0 1 9>,
 		 <&cpm_syscon0 1 6>, <&cpm_syscon0 1 5>;
-- 
1.7.9.5

